gdx/src/com/badlogic/gdx/assets/AssetManager.java

```diff
@@ -127,13 +129,13 @@ public AssetManager (FileHandleResolver resolver) {
 		if (asset == null) throw new GdxRuntimeException("Asset not loaded: " + fileName);
 		return asset;
 	}
-	
+
```

Comment:
All these formatting changes are unnecessary, are you using the [Eclipse formatter](https://github.com/libgdx/libgdx/blob/master/eclipse-formatter.xml)?